articles/cosmos-db/sql/sql-query-is-number.md

```diff
@@ -37,17 +37,18 @@ IS_NUMBER(<expr>)
 SELECT   
     IS_NUMBER(true) AS isNum1,   
     IS_NUMBER(1) AS isNum2,  
-    IS_NUMBER("value") AS isNum3,   
-    IS_NUMBER(null) AS isNum4,  
-    IS_NUMBER({prop: "value"}) AS isNum5,   
-    IS_NUMBER([1, 2, 3]) AS isNum6,  
-    IS_NUMBER({prop: "value"}.prop2) AS isNum7  
+    IS_NUMBER("value") AS isNum3, 
+    IS_NUMBER("12") AS isNum4,
+    IS_NUMBER(null) AS isNum5,  
+    IS_NUMBER({prop: "value"}) AS isNum6,   
+    IS_NUMBER([1, 2, 3]) AS isNum7,  
+    IS_NUMBER({prop: "value"}.prop2) AS isNum8  
 ```  
   
  Here is the result set.  
   
 ```json
-[{"isNum1":false,"isNum2":true,"isNum3":false,"isNum4":false,"isNum5":false,"isNum6":false,"isNum7":false}]  
+[{"isNum1":false,"isNum2":true,"isNum3":false,"isNum4":false,"isNum5":false,"isNum6":false,"isNum7":false,"isNum8":false}]  
```

Comment:
```suggestion
[
    {
        "isBooleanANumber": false,
        "isNumberANumber": true,
        "isTextStringANumber": false,
        "isNumberStringANumber": false,
        "isNullANumber": false,
        "isObjectANumber": false,
        "isArrayANumber": false,
        "isObjectStringPropertyANumber": false,
        "isObjectNumberPropertyANumber": true
    }
]
```

If you have no personal objections, I would like to prettify the code output a bit to make it more human readable.